lib/v2/datasource/remote/api/signup_repository.dart

```diff
@@ -69,4 +75,37 @@ class SignupRepository extends NetworkRepository {
             }))
         .catchError((error) => mapHttpError(error));
   }
+
+  Future<Result> createAccount({String? accountName, String? inviteSecret, String? displayName}) async {
+    EOSPrivateKey privateKey = EOSPrivateKey.fromRandom();
+    EOSPublicKey publicKey = privateKey.toEOSPublicKey();
+
+    final applicationAccount = Config.onboardingAccountName;
+
+    final actions = <Action>[
+      Action()
+        ..account = applicationAccount
+        ..name = 'acceptnew'
+        ..authorization = <Authorization>[
+          Authorization()
+            ..actor = applicationAccount
+            ..permission = 'application'
+        ]
```

Comment:
Could you take a look on the NetworkReposirtory we save things like this action name and permissions types there. In order to avoid have here hardcoded strings.

Reply:
IMO this is fine as it is - it's a chain transaction so it should have all the data specified in one place, ie here.

Unless there's a generator for actions there. Still any of these could be different for any action, so I prefer this - just fill all the fields for the action and keep it very explicit, what the permission is, what the account is, what the action is, and what the data is.

Reply:
Please follow what we currently have in place, 
If we must change this, we can. I am not opposed. But until we change them all, i rather have consistency.

I rather have consistency across the application.

Reply:
@n13 I understand and it makes sense, it is visually clearer to see the action and not have to go to _see implementation_, as well as It is possible to vary names for an action or permission, but I agree with Gery, it would be to change what we already have advanced. and at least in this way we seek not to repeat these strings and to have all in one place.

Of cuerse, If it is very difficult to keep the same we can change it later.

Reply:
Addressed

Reply:
> Please follow what we currently have in place,
> If we must change this, we can. I am not opposed. But until we change them all, i rather have consistency.
> 
> I rather have consistency across the application.

So this is just about using constants? In this case I agree completely.